srcpkgs/LuaJIT/template

```diff
@@ -1,71 +1,58 @@
 # Template file for 'LuaJIT'
 pkgname=LuaJIT
-version=2.1.0beta3
-revision=2
-_so_version=2.1.0
-_dist_version=${_so_version}-beta3
+version=2.1.1692580715
+revision=1
+_dist_version=2.1.ROLLING
```

Comment:
Upstream merges into this branch up to few times a day, what means checksum changes before we accept PR. Use commit hash instead.

Reply:
I am *not* using the branch ref. I am using the *tag* ref that was updated 5 months ago at the start of the rolling release transition: https://repo.or.cz/luajit-2.0.git/refs

should I still use commit hash?

Reply:
Anytime upstream moves tag/branch/whatever, template breaks.  Please use commit hash so we can have rebuilds.

Reply:
alrighty. might as well move to the actual rolling release branch then